.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -181,3 +181,7 @@ if [ ${SELINUX_PERMISSIVE} == "yes" ]; then
 	# The new SELinux configuration will trigger another reboot.
 	wait_for_reboot
 fi
+
+# FIXME: Remove when https://github.com/kata-containers/kata-containers/pull/8417 is resolved
+# Selinux context is currently not handled by kata-deploy
+oc apply -f ${deployments_dir}/relable_selinux.yaml
```

Comment:
should it be relabel_selinux instead of **relable**?

Here you also will need to check the pod is running to ensure the relabel operation succeeded.

Reply:
Hmm, it's true we should perhaps wait for a message of completion before proceeding. As for the successfullness I don't think that's a good idea as we don't know it's a selinux capable system.